indexer-app/test/cmr/indexer/test/data/concepts/association_util.clj

```diff
@@ -9,16 +9,25 @@
 
   (testing "Tests that building a concept named association list from a 
             generics association list works."
-    (let [generic-associations ["DQS1200000012-PROV1"
-                                "DQS1200000013-PROV1"
-                                "OO1200000014-PROV1"
-                                "SO1200000015-PROV1"
-                                "SO1200000016-PROV1"
-                                "SE1200000017-PROV1"
-                                "GRD1200000018-PROV1"]
-          expected {:dataqualitysummaries ["DQS1200000012-PROV1" "DQS1200000013-PROV1"]
-                    :orderoptions ["OO1200000014-PROV1"]
-                    :serviceoptions ["SO1200000015-PROV1" "SO1200000016-PROV1"]
-                    :serviceentries ["SE1200000017-PROV1"]
-                    :grids ["GRD1200000018-PROV1"]}]
-      (is (= expected (assoc-util/generic-assoc-list->assoc-struct generic-associations))))))
+    (let [generic-associations [{:source-concept-identifier "C1200000021-PROV1"
+                                 :associated-concept-id "DQS1200000012-PROV1"
+                                 :data {:hello "ok"}}
+                                {:source-concept-identifier "C1200000021-PROV1"
+                                 :associated-concept-id "DQS1200000013-PROV1"}
+                                {:source-concept-identifier "C1200000021-PROV1"
+                                 :associated-concept-id "OO1200000014-PROV1"}
+                                {:source-concept-identifier "C1200000021-PROV1"
+                                 :associated-concept-id "SO1200000015-PROV1"}
+                                {:source-concept-identifier "SO1200000016-PROV1"
+                                 :associated-concept-id "C1200000021-PROV1"}
+                                {:source-concept-identifier "C1200000021-PROV1"
+                                 :associated-concept-id "SE1200000017-PROV1"}
+                                {:source-concept-identifier "C1200000021-PROV1"
+                                 :associated-concept-id "GRD1200000018-PROV1"}]
```

Comment:
Could you add revision-ids in the source and associated concepts?

Reply:
they have been added.